packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -249,15 +276,8 @@ let installHandlers = () => {
   installHandlers = () => { }; // Done doing stuff!
 }
 
-export function isTrackedEvent(eventName: string): boolean {
-  return TrackedEvents.has(eventName);
-}
-
-export function trackInteractable(eventName: string): boolean {
-  installHandlers();
-  const alreadyTracked = TrackedEvents.has(eventName);
-  TrackedEvents.add(eventName);
```

Comment:
This mutates the set of events (potentially) and doesn't have all the handlers needed for the map/set to be combined.  

The tests still pass and the data-*able attributes are still set since install handlers is called in enableUIEventHandlers.  Open to other thoughts but the last suggestion was to combine these objects tracking events that are registered and this part was causing some complications with using the shared map when it doesn't have the necessary value object to set